src/data/enemies/boss55-ert.js

```diff
@@ -0,0 +1,198 @@
+const BOSSES = {
```

Comment:
I haven't looked at the exact stats but it looks like you split the difference between this and the other boss so that's good!